src/app/components/profile/cardDetails/profileView.tsx

```diff
@@ -23,15 +16,16 @@ type Props = {
 const ProfileView = ({data}: Props) => {
   return (
     <View style={styles.profileContainer}>
-      {data.map((item, index) =>
-        item.uri ? (
-          <CircleView
-            key={index}
-            data={{name: item.name}}
-            uri={item.uri}
-            handlePress={handlePress}
-          />
-        ) : undefined,
+      {data.map(
+        (item, index) =>
```

Comment:
we can destructure name and uri from item
```suggestion
        ({name, uri}, index) =>
```